docs/docs/cmd/spo/list/list-get.md

```diff
@@ -22,6 +22,9 @@ m365 spo list get [options]
 `-p, --properties [properties]`
 : Comma-separated list of properties to retrieve from the list. Will retrieve all properties possible from default response, if not specified.
 
+`--withPermissions`
+: Set if you want to return associated roles and permissions (associatedOwnerGroup, associatedMemberGroup and associatedVisitorGroup) of the list.
```

Comment:
Aren't the associated groups something for a web rather than a list? Looking at the data that we're retrieving, shouldn't we say that we retrieve information about unique permissions and permissions assignments?